ctx.go

```diff
@@ -1414,48 +1415,160 @@ func (c *DefaultCtx) Send(body []byte) error {
 	return nil
 }
 
-var (
-	sendFileOnce    sync.Once
-	sendFileFS      *fasthttp.FS
-	sendFileHandler fasthttp.RequestHandler
-)
+// SendFile defines configuration options when to transfer file with SendFileWithConfig.
+type SendFile struct {
```

Comment:
pls move the struct to the top or in a different file togehter with the other logic